### ISSUE TITLE
csg_stat_angular throws std::bad_alloc

### ISSUE THREAD

**Issue by junghans** [votca/csg#294]
Testing 1.5_rc1 on Fedora, I get a:
```
48/74 Test #48: Run_csg_stat_angular ...................Child aborted***Exception:   0.18 sec
begin to calculate distribution functions
# of bonded interactions: 0
# of non-bonded interactions: 1
I have 648 beads in 216 molecules
I have 216 beads in 216 molecules for the coarsegraining
Reading frame, timestep 0
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
```

`std::bad_alloc` usually means the system ran out of heap memory, but the test system is so small that 
should not be the case!

Any ideas?

**Comment by JensWehner** [comment]
hmm... maybe some runaway access. My big guess is a massive memory leak.

Valgrind will help.

**Comment by JensWehner** [comment]
how do I execute this test on its own?

**Comment by junghans** [comment]
`ctest -R Run_csg_stat_angular`

**Comment by JensWehner** [comment]
is this fixed?

**Comment by junghans** [comment]
@jens, I don't think so.

**Comment by junghans** [comment]
Seems to be fixed by votca/tools#90
